content/md/what-is/what-is-an-asset.md

```diff
@@ -0,0 +1,53 @@
+---
+id: what-is-an-asset
+themes: first-steps, akeneo-concepts, manage-your-assets
+title: What is an **asset**?
+popular: false
+ee-only: true
+related: assets-transformation, upload-assets, work-with-assets
+---
+
+# Definition of an asset
+
+An asset can be a HD image, a video, a PDF installation guide... Any kind of files that can be used to enrich a product in Akeneo.
+
+::: ee
+Assets are only available in the Enterprise Edition.
+:::
+
+An asset is always `scopable` because an asset reference file can be declined for each channel depending on your export needs.  
+It can also be `localisable`, if you want to have a specific version for the locales enabled in the PIM. For instance, you can have a PDF guide in English, another one in French, another one in German...
+Thus, for each asset created, you may have several files attached to it.
+
+![Asset](../img/Settings_What-is-an-asset.svg)
+
+An asset has a unique code which is mandatory and optional properties:
```

Comment:
```suggestion
An asset has a unique code which has mandatory and optional properties:
```

Reply:
I guess it was missing a comma here.
We wanted to say:
```suggestion
An asset has a unique code which is mandatory, and optional properties:
```